src/pyastar2d/astar_wrapper.py

```diff
@@ -19,14 +19,30 @@
     ctypes.c_int,   # start index in flattened grid
     ctypes.c_int,   # goal index in flattened grid
     ctypes.c_bool,  # allow diagonal
+    ctypes.c_int,   # heuristic_override
+    ctypes.c_float, # tiebreaker_coefficient
 ]
 
 
 def astar_path(
         weights: np.ndarray,
         start: Tuple[int, int],
         goal: Tuple[int, int],
-        allow_diagonal: bool = False) -> Optional[np.ndarray]:
+        allow_diagonal: bool = False,
+        heuristic_override: int = 0,
```

Comment:
From the Python API the user should be able to pass a `str` indicating which heuristic function they want. See another comment.

Reply:
Created intEnum